src/core-skills/bmad-brainstorming/scripts/tests/test_brain.py

```
@@ -0,0 +1,164 @@
+def test_json_output(lib, capsys):
+    import json
+    brain.main(["--file", str(lib), "--json", "categories"])
+    data = json.loads(capsys.readouterr().out)
+    assert {"category": "wild", "count": 2} in data
+
+
+def test_random_respects_n_and_category(lib, capsys):
+    brain.main(["--file", str(lib), "random", "--category", "wild", "-n", "5"])
+    lines = capsys.readouterr().out.strip().splitlines()
+    assert len(lines) == 2  # only 2 wild exist, n capped
+    assert all(l.startswith("wild\t") for l in lines)
```

Ruff (0.15.14)

[error] Ambiguous variable name: `l`

(E741)